database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -16,6 +16,7 @@ public function up()
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
             $table->string('name');
+            $table->string('password');
```

Comment:
There should not be a password for users, because when user try to login we send a request to partners service, sending user and password. This pasword can be changed within their project. We will not check password from our database.